practitioner-service/src/main/resources/application-local.yml

```diff
@@ -26,3 +26,12 @@ config:
   - name: prefix
     displayName: Prefix
     validationRegex: "^[a-zA-Z]{0,35}$"
+
+---
+site-service:
+  ribbon:
+    listOfServers: localhost:8083
```

Comment:
We can simplify the docker-compose if we use the same value it already inject like this example:
_${site.service.uri:http://localhost:8083}_

Reply:
Done